data/yam/agama/auto/alp-tw.sh

```diff
@@ -0,0 +1,8 @@
+set -ex
+
+/usr/bin/agama config set software.product=Tumbleweed
```

Comment:
we agreed to use a different format of .sh for each product, please pick json or jsonnet so we can cover all of them with the automation.

Reply:
let's delete the file, we want to run the same scenario for both archs.

Reply:
why need delete the file? And this file should be used for both archs.

Reply:
because we talked offline that we will use for all arch installing the same product the same file and we will use different format for other products, we used for installing Bedrock with Agama .sh, so we don't repeat the same here, we use jsonnet or json. We don't need the very same scenario using json and using the script, they are functionally the same.